aws/mgmt.tpl

```diff
@@ -1,65 +0,0 @@
-#These need to come from terraform
```

Comment:
this file has been mentioned in readme.  Can we change it to pint aws/tkg_vpc/templates/mgmt.tpl

r config file(**mgmt.tpl**) to integrate Pinniped with external IDP.

Reply:
great find. i've updated this to reference `/tkg_vpc/templates/mgmt.tpl`

i left off the `aws` in the path as the readme is in the aws folder so kept it relative to that.

if you want that adjusted let me know.